TAJO-1462: Replace CSV examples into TEXT examples in docs

dongjoon-hyun opened the pull request.

jihoonson commented:
Thanks for your contribution. I have two comments. 
- If any options are given, the delimiter of text files is ','. I think that we need at least one example without the delimiter option. 
- You modified the document for the csv file. However, that document still looks to explain how to use csv files. So, it would be better to change the document title together.

dongjoon-hyun commented:
@jihoonson . Thank you! First, I will omit " text.delimiter=',' " and make some notes for the default value. Second, I will replace csv.rst into text.rst. Do you think that is enough?

jihoonson commented:
Right. It will be nice.

dongjoon-hyun commented:
Hi, @jihoonson . I rebased and changed. But I need your guide for one thing. In the document, I found that Tajo defines CSV files as 'character-separated values' files, not 'comma-separated values' file. In that case, I think CSV seems to be a more specific and exact term than TEXT. As a result, `csv.rst` looks good to me. How do you think about this?  (By the way, I found that the default value is explained enough in the doc.)

jihoonson commented:
In that case, users may be confused because we also provide the ```csv``` file format. 
I think that it will be enough if we introduce the ```text``` file as the delimiter-separated values format.

dongjoon-hyun commented:
I see, @jihoonson . You're right. I renamed and changed more description according to your comments.

jihoonson commented on tajo-docs/src/main/sphinx/table_management/text.rst:
```diff
@@ -1,21 +1,21 @@
 *************************************
 
-A character-separated values (CSV) file represents a tabular data set consisting of rows and columns.
+A character-separated values text file represents a tabular data set consisting of rows and columns.
```
It would be better if readers can easily understand what text means. As you know, ```text``` can represent both the TEXT file and strings.
How about emphasize the letters ```text``` when it means the TEXT file? For sphyinx syntax, please refer to http://sphinx-doc.org/rest.html.

dongjoon-hyun replied:
Sure. That would be better.

jihoonson commented on tajo-docs/src/main/sphinx/table_management/text.rst:
```diff
@@ -1,21 +1,21 @@
 
 In order to specify a certain file format for your table, you need to use the ``USING`` clause in your ``CREATE TABLE``
-statement. The below is an example statement for creating a table using CSV files.
+statement. The below is an example statement for creating a table using text files.
```
Same comment with the above one.

dongjoon-hyun commented:
Rebased.

jihoonson commented on tajo-docs/src/main/sphinx/table_management/text.rst:
```diff
@@ -34,19 +34,19 @@ Physical Properties
+* ``text.null``: ``NULL`` character. The default ``NULL`` character is an empty string ``''``. Hive's default ``NULL`` character is ``'\\N'``.
 * ``compression.codec``: Compression codec. You can enable compression feature and set specified compression algorithm. The compression algorithm used to compress files. The compression codec name should be the fully qualified class name inherited from `org.apache.hadoop.io.compress.CompressionCodec <https://hadoop.apache.org/docs/current/api/org/apache/hadoop/io/compress/CompressionCodec.html>`_ ...
-* ``csvfile.serde`` (deprecated): custom (De)serializer class. ``org.apache.tajo.storage.TextSerializerDeserializer`` is the default (De)serializer class.
+* ``text.serde``: custom (De)serializer class. ``org.apache.tajo.storage.TextSerializerDeserializer`` is the default (De)serializer class.
```
The default SerDe class is ```org.apache.tajo.storage.text.CSVLineSerDe```.

dongjoon-hyun replied:
Of course! Thank you again, @jihoonson !

jihoonson commented:
+1. The patch LGTM in overall.
I left one comment on the wrong name of the default SerDe class.
I'll fix it before commit.

dongjoon-hyun commented:
Thank you, @jihoonson !
